easybuild/easyconfigs/r/R/R-3.3.3-intel-2017a-X11-20170314.eb

```diff
@@ -0,0 +1,510 @@
+    ('abc.data', '1.0', ext_options),
+    ('abc', '2.1', ext_options),
+    ('lhs', '0.14', ext_options),
+    ('tensorA', '0.36', ext_options),
+    ('EasyABC', '1.5', ext_options),
+    ('shape', '1.4.2', ext_options),
+    ('whisker', '0.3-2', ext_options),
+    ('rstudioapi', '0.6', ext_options),
+    ('commonmark', '1.2', ext_options),
+    ('desc', '1.1.0', ext_options),
+    ('roxygen2', '6.0.1', ext_options),
+    ('git2r', '0.18.0', ext_options),
+    ('rversions', '1.0.3', ext_options),
+    ('withr', '1.0.2', ext_options),
+    ('devtools', '1.12.0', ext_options),
+    ('Rook', '1.1-1', ext_options),
+    ('Cairo', '1.5-9', dict(ext_options.items() + [('patches', ['Cairo-1.5-9.patch'])])),
+    ('RMTstat', '0.3', ext_options),
+    ('Lmoments', '1.2-3', ext_options),
+    ('distillery', '1.0-2', ext_options),
+    ('extRemes', '2.0-8', ext_options),
+    ('pixmap', '0.4-11', ext_options),
+    ('tkrplot', '0.0-23', ext_options),
+    ('misc3d', '0.8-4', ext_options),
+    ('multicool', '0.1-10', dict(ext_options.items() + [('patches', ['multicool-0.1-10_icpc-wd308.patch'])])),
+    ('ks', '1.10.5', ext_options),
+    ('logcondens', '2.1.5', ext_options),
+    ('Iso', '0.0-17', ext_options),
+    ('penalized', '0.9-50', ext_options),
+    ('clusterRepro', '0.5-1.1', ext_options),
+    ('randomForestSRC', '2.4.2', ext_options),
+    ('sm', '2.2-5.4', ext_options),
+    ('psych', '1.6.12', ext_options),
+    ('pbivnorm', '0.6.0', ext_options),
+    ('lavaan', '0.5-23.1097', ext_options),
+    ('matrixcalc', '1.0-3', ext_options),
+    ('arm', '1.9-3', ext_options),
+    ('mi', '1.0', ext_options),
+    ('visNetwork', '1.0.3', ext_options),
+    ('rgexf', '0.15.3', ext_options),
+    ('influenceR', '0.1.0', ext_options),
+    ('DiagrammeR', '0.9.0', ext_options),
+    ('sem', '3.1-8', ext_options),
+    ('jpeg', '0.1-8', ext_options),
+    ('network', '1.13.0', ext_options),
+    ('statnet.common', '3.3.0', ext_options),
+    ('sna', '2.4', ext_options),
+    ('glasso', '1.8', dict(ext_options.items() + [('patches', [('glasso-1.8-ifort-no-fixed.patch', 1)])])),
+    ('huge', '1.2.7', ext_options),
+    ('d3Network', '0.5.2.1', ext_options),
+    ('ggm', '2.3', ext_options),
+    ('qgraph', '1.4.2', ext_options),
+    ('diveRsity', '1.9.89', ext_options),
+    ('doSNOW', '1.0.14', ext_options),
+    ('phangorn', '2.1.1', ext_options),
+    ('geepack', '1.2-1', ext_options),
+    ('lubridate', '1.6.0', ext_options),
+    ('biom', '0.3.12', ext_options),
+    ('pim', '2.0.0.2', ext_options),
+    ('minpack.lm', '1.2-1', ext_options),
+    ('rootSolve', '1.7', ext_options),
+    ('diagram', '1.6.3', ext_options),
+    ('FME', '1.3.5', ext_options),
+    ('bmp', '0.2', ext_options),
+    ('readbitmap', '0.1-4', ext_options),
+    ('purrr', '0.2.2', ext_options),
+    ('downloader', '0.4', ext_options),
+    ('imager', '0.40.1', dict(ext_options.items() + [('patches', ['imager-0.40.1_fix-CImg-icpc.patch'])])),
```

Comment:
@boegel At the end it makes sense. I can do it when I am back from hollidays. (18th April 2017).